### PR TITLE
Why it’s critical to get environment promotion right

### PR DESCRIPTION
**Pull request by ellapradella**
# About

This is a post about...

# Release

Is this post for a specific release? If so, which one?

# Blog image idea

The design team will work on a design for the blog image. Add any ideas you have about the graphic image here. (Tegan will request the image from design and add it to the post when it's ready.)

# Before you submit your post

Check everything off this list before you submit your post for review. 

- [ ] I'm familiar with the [Octopus voice and style](https://www.octopus.design/932c0f1a9/p/13df3a-voice-and-style), the [guide for blog content basics](https://www.octopus.design/932c0f1a9/p/901d2a-blog-content), and [writing educational content](https://www.octopus.design/932c0f1a9/p/137b06-educational-content---how-we-write-about-what-we-do).
- [ ] I've checked the content using Grammarly (Octopus has a corporate license) and the [Hemingway app](https://hemingwayapp.com/) to remove typos, grammatical errors, and passive voice. 
- [ ] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://www.octopus.design/932c0f1a9/p/901d2a-blog-content/t/817249) and none of those apply.
- [ ] The images I've included follow the [rules for working with images](https://www.octopus.design/932c0f1a9/p/5061d7-working-with-images).
- [ ] I've added alt text for all the images I've included (alt text describes an image to people unable to see it, 125 characters max)
- [ ] The draft is complete and this post is ready to be reviewed. (Please mark the post 'ready for review' at the bottom of the PR.)
- [ ] I've dropped a note in Slack in #topic-content-marketing to let Tegan Ali know my post is ready for review. (No need to fill out a content brief.)
- [ ] I'll draft social media posts for Twitter and LinkedIn. I'll post them for review in #topic-social-media after the post has been edited or after the PR has been merged.

## How to review this PR

If there's anything you'd like reviewers to focus on, mention it here.

## Publication date

If there are considerations for when to publish this post, mention them here. For example, this post is supporting material for a webinar I'll be conducting on date, or this post should not published until after a specific release on date.

### CODE REVIEW

File: blog/2025-01/get-environment-promotion-right/index.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+- **Reduced risk of something going wrong** - The more you test a new version of an application before it goes to production, the less risk there is that something will go wrong and create downtime. Environment promotion includes testing and gates to reduce the risk of unexpected errors when it reaches customers.
+- **Crisis management** - Being confident in your deployment processes also helps during emergencies. Relying on a trusted, well-tested, automated process is much better than trying to improvise. Using an improvised process in a high-pressure situation makes you more likely to introduce even more problems.
+
+## Automating environment promotion with Continuous Delivery
+
+Every software company has some form of environment and rules around promotion in place. Yet, environment promotion reamins a challenge. 
+
+One of the main benefits of Continuous Delivery (CD) and deployment automation is that it helps you increase deployment frequency, which helps you improve DORA metrics, as all focus on the quality of your environment promotion solution.
```
Comment (TeganAli) [inline blog/2025-01/get-environment-promotion-right/index.md:70]:
```suggestion
Continuous Delivery (CD) and deployment automation help you increase deployment frequency and the quality of your promotions between environments. This, in turn, helps you improve your DORA metrics. Teams that perform well against DORA metrics are more likely to achieve better customer satisfaction, operational efficiency, and overall organizational performance.
```

File: blog/2025-01/get-environment-promotion-right/index.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ Having confidence in what you released is important for a few reasons:
 
 Every software company has some form of environment and rules around promotion in place. Yet, environment promotion reamins a challenge. 
 
-One of the main benefits of Continuous Delivery (CD) and deployment automation is that it helps you increase deployment frequency, which helps you improve DORA metrics, as all focus on the quality of your environment promotion solution. 
+Continuous Delivery (CD) and deployment automation help you increase deployment frequency and the quality of your promotions between environments. This, in turn, helps you improve your DORA metrics. Teams that perform well against DORA metrics are more likely to achieve better customer satisfaction, operational efficiency, and overall organizational performance.
```
Comment (TeganAli) [inline blog/2025-01/get-environment-promotion-right/index.md:70]: @nikita-dergilev can you please check you're happy with this change?

File: blog/2025-01/get-environment-promotion-right/index.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+- **Reduced risk of something going wrong** - The more you test a new version of an application before it goes to production, the less risk there is that something will go wrong and create downtime. Environment promotion includes testing and gates to reduce the risk of unexpected errors when it reaches customers.
+- **Crisis management** - Being confident in your deployment processes also helps during emergencies. Relying on a trusted, well-tested, automated process is much better than trying to improvise. Using an improvised process in a high-pressure situation makes you more likely to introduce even more problems.
+
+## Automating environment promotion with Continuous Delivery
+
+Every software company has some form of environment and rules around promotion in place. Yet, environment promotion reamins a challenge.
```
Comment (TeganAli) [inline blog/2025-01/get-environment-promotion-right/index.md:68]:
```suggestion
Every software company has some form of environment and rules around promotion in place. Yet, environment promotion remains a challenge. 
```

File: blog/2025-01/get-environment-promotion-right/index.md (hunk 2 above)
Comment (nikita-dergilev) [reply blog/2025-01/get-environment-promotion-right/index.md:70]: I think we can say that 'The best way to measure it comprehensively is DORA metrics'. Metrics are not a thing by itself.

I like how you made it clearer overall! Thanks!

**Review by TeganAli — APPROVED** [review]

**Pull request merged**